code/modules/holiday/holidays.dm

```diff
@@ -640,3 +640,13 @@
 
 /datum/holiday/hebrew/passover/getStationPrefix()
 	return pick("Matzah", "Moses", "Red Sea")
+
+/datum/holiday/pride_month
+	name = "Pride Month"
+	begin_day = 1
+	begin_month = JUNE
+	end_day = 30
```

Comment:
make it the length of the previous pride week, since we don't usually have events longer than a week (except maybe christmas iirc)

Reply:
done